### PR TITLE
Websocket client

### PR DESCRIPTION
**Pull request by dmanchon**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,6 +38,6 @@
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/vinissimus/async-asgi-testclient",
-    version="0.2.2",
+    version="0.2.3",
```
Comment (masipcat) [inline setup.py]: You can update to 1.0.0

**Comment by dmanchon** [comment]
> Can you write a test?
yeah, i need to read how ws works both on starlette and quart first, i can try to do it tonight after work.

Also after guillotina merge the asgi support it will be great to add it to the tests too, great opportunity to define a minimal guillotina app.

**Comment by masipcat** [comment]
> > Can you write a test?
> > yeah, i need to read how ws works both on starlette and quart first, i can try to do it tonight after work.
> 
> Also after guillotina merge the asgi support it will be great to add it to the tests too, great opportunity to define a minimal guillotina app.

totally agree! :)

**Comment by masipcat** [comment]
and can you update the "websocket support" checkbox in [README](https://github.com/vinissimus/async-asgi-testclient#supports) as well?

**Comment by masipcat** [comment]
Thank you very much!

**Comment by masipcat** [comment]
v1.0.0 released

**Comment by matthewchung74** [comment]
Thanks for writing this!

**Pull request merged**